### PR TITLE
Dns aliases

### PR DESCRIPTION
**Pull request by yaronMSFT**
Adding aliases

### CODE REVIEW

**Review by oshezaf — APPROVED** [review]

**Review by oshezaf — CHANGES_REQUESTED** [review]
I think you did not update version numbers

**Review by oshezaf — DISMISSED** [review]

File: Parsers/ASimDns/ProductParsers/ASimDnsMicrosoftSysmon.yaml [hunk 1]
```diff
@@ -98,7 +98,9 @@ ParserQuery: |
           Domain = DnsQuery,
           Dvc = DvcHostname,
           SrcHostname = DvcHostname,
+          Hostname=DvcHostname,
           Src = DvcHostname,
+          //IpAddr=SrcIpAddr,
```
Comment (oshezaf) [inline Parsers/ASimDns/ProductParsers/ASimDnsMicrosoftSysmon.yaml:103]: Why commented out?

File: Parsers/ASimDns/ProductParsers/vimDnsMicrosoftSysmon.yaml [hunk 2]
```diff
@@ -186,6 +186,8 @@ ParserQuery: |
        Dvc = DvcHostname,
        SrcHostname = DvcHostname,
        Src = DvcHostname,
+       // IpAddr=SrcIpAddr,
```
Comment (oshezaf) [inline Parsers/ASimDns/ProductParsers/vimDnsMicrosoftSysmon.yaml:189]: Why commented out?

File: Parsers/ASimDns/ProductParsers/vimDnszScalerZIA.yaml [hunk 3]
```diff
@@ -67,7 +67,7 @@ ParserQuery: |
         and (array_length(response_has_any_prefix) == 0 or has_any_ipv4_prefix(DeviceCustomString6, response_has_any_prefix))
         and (responsecodename in ('*', 'NOERROR') or DeviceCustomString6 =~ responsecodename)) // NOERROR is determined only later
     | extend
-        EventResultDetails = iff (DeviceCustomString6 matches regex '^\\d', 'NOERROR', DeviceCustomString6)
+        EventResultDetails = iff (DeviceCustomString6 matches regex @'^([A-Z_]+)$', DeviceCustomString6, 'NOERROR')
```
Comment (oshezaf) [inline Parsers/ASimDns/ProductParsers/vimDnszScalerZIA.yaml:70]: Can you remind me what the issue was? Don't we need a lookup?

**Review by oshezaf — CHANGES_REQUESTED** [review]

File: Parsers/ASimDns/ProductParsers/ASimDnsCorelightZeek.yaml [hunk 4]
```diff
@@ -57,15 +57,20 @@ ParserQuery: |
...
     // Aliases
     | extend 
         DnsResponseCodeName=EventResultDetails, 
         Domain=DnsQuery,
         IpAddr=SrcIpAddr,
         Src=SrcIpAddr,
         Dst=DstIpAddr,
-        Duration=DnsNetworkDuration
+        Duration=DnsNetworkDuration,
+        SrcHostname=SrcIpAddr,
```
[the remark this answers is not in the document]
Comment (yaronMSFT) [reply Parsers/ASimDns/ProductParsers/ASimDnsCorelightZeek.yaml:72]: removed. See onenote comment

**Pull request merged**